examples/chip-tool/commands/common/ModelCommand.h

```diff
@@ -58,6 +63,7 @@ class ModelCommand : public NetworkCommand
     void ParseReadAttributeResponseCommandSuccess(uint16_t clusterId, uint16_t attrId, uint8_t * message,
                                                   uint16_t messageLen) const;
     void ParseReadAttributeResponseCommandFailure(uint16_t clusterId, uint16_t attrId, uint8_t status, uint16_t messageLen) const;
+    void ParseIdentifyClusterCommand(uint16_t endpointID, uint8_t * message, uint16_t messageLen) const;
```

Comment:
Why is this here?  It's not used, right?

Reply:
Sorry this was a left over code from an earlier version of the PR that I hadn't posted.